src/server.ts

```diff
@@ -1,28 +1,34 @@
 import bodyParser from "body-parser";
-import express, {Express} from "express";
+import express, { Express } from "express";
 import mongoose from "mongoose";
 import authRoutes from "./routes/auth.route";
 import usersRoutes from "./routes/users.route";
+import recommendationRoutes from "./routes/recommendation.route";
+import matchRoutes from "./routes/match.route";
 import dotenv from "dotenv";
 import swaggerJsDoc from "swagger-jsdoc";
 import swaggerUI from "swagger-ui-express";
+import cors from "cors";
+import { handleErrorMiddleware } from "./middlewares/error.middleware";
+import router from "./routes/file.route";
 
 dotenv.config();
 const app = express();
 app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({extended: true}));
+app.use(cors());
+app.use(bodyParser.urlencoded({ extended: true }));
 
 const options = {
-    definition: {
-        openapi: "3.0.0",
-        info: {
-            title: "Assignment Two REST API",
-            version: "1.0.0",
-            description: "REST server including authentication using JWT",
-        },
-        servers: [{url: "http://localhost:3000"}],
+  definition: {
+    openapi: "3.0.0",
+    info: {
+      title: "Assignment Two REST API",
```

Comment:
Change to Sport Scanner REST API
```suggestion
      title: "Sport Scanner REST API",
```